### ISSUE TITLE
German umlauts not working

### ISSUE THREAD

**Issue by DirkMahler** [swt14w01/JRMDS#9]
After entering a tag containing german umlaut "ü" there are strange characters while rendering it in lists and forms

**Comment by HandBreit** [comment]
Is this done? It works...

**Comment by poisonnuke** [comment]
not really... try to enter some umlaut's in any form field and then update the regarding component...
